app/views/accounts/show.html.erb

```diff
@@ -24,9 +24,8 @@
                    :include_blank => t("auth.providers.none"),
                    :hide_label => true,
                    :wrapper => { :class => "col-auto mb-0" } %>
-      <%= f.text_field(:auth_uid, :hide_label => true, :wrapper => { :class => "col mb-0" }) %>
+      <%= f.hidden_field(:auth_uid, :hide_label => true, :wrapper => { :class => "col mb-0" }) %>
```

Comment:
I suspect we don't actually need this field any more - it was only there to allow the UID to be edited when OpenID was in use and for other methods users shouldn't be changing the UID so I think just leaving it out will be fine.

We would need to change the `update` method in `AccountsController` to stop comparing the uid for equality and then including it in the redirect but we probably want to do the second part anyway as it's only there for openid.